openadapt/splashscreen.py

```diff
@@ -0,0 +1,102 @@
+import sys , multiprocessing , threading , time
+
+from PySide6 import QtCore, QtWidgets, QtGui 
+from PySide6.QtWidgets import QApplication  ,  QSplashScreen , QMainWindow , QLabel
+from PySide6.QtGui import QPixmap , QGuiApplication
+from loguru import logger
+
+queue = multiprocessing.Queue()
+label_2 = False
+label_3 = False
+
+def update_progress(num):
+    try:
+        global queue
+        queue.put(num)
+    except Exception as e : 
+        logger.error(" Error in updating progress : "+ str(e) )
+
+
+def update_label():
```

Comment:
@Jagdeepkhunan please paste your code into ChatGPT and ask it to explain what is wrong with it and/or refactor it. There are numerous issues related to variable naming and general code organization.